salt/returners/influxdb_return.py

```diff
@@ -331,3 +335,55 @@ def prep_jid(nocache=False, passed_jid=None):  # pylint: disable=unused-argument
     Do any work necessary to prepare a JID, including sending a custom id
     '''
     return passed_jid if passed_jid is not None else salt.utils.jid.gen_jid(__opts__)
+
+
+def event_return(events):
+    '''
+    Return a list of events to InfluxDB
+
+    This is largely copied from the mysql event_return function and reading over
+    the following documentation:
+    https://docs.saltstack.com/en/latest/ref/returners/index.html#event-returners
+    https://docs.saltstack.com/en/latest/ref/returners/index.html#event-support
+    https://github.com/influxdata/influxdb-python
+
+    This was tested with pip module influxdb 5.2.0 installed in a Docker
+    container against InfluxDB 1.6.0 (Docker version)
+
+
+    It should be noted that this code is not very robust and is just a start to
+    get this working.
+    '''
+
+    client = _get_serv()
+
+    for event in events:
+        # This should probably be redone to join all the events together
+        # and then send them all at once as long as the server can handle it?
+        # This method may require more handling splitting the data into more
+        # manageable chunks for sending.
+        req = [
+            {
+                'measurement': 'events',
+                'tags': {
+                    'master_id': __opts__.get('id', ''),
+                    'fun': salt.utils.json.dumps(event['data'].get('fun', '')),
+                    'minions': salt.utils.json.dumps(event['data'].get('minions', '') or event['data'].get('id', '')),
```

Comment:
@skizunov Thanks for the suggestions. I will make those recommended changes. 

I also need to update it to get it to group the events together to make a single insert instead of one insert per event.